mon-pix/app/components/authentication/signup-form/signup-form-validation.js

```diff
@@ -0,0 +1,102 @@
+import { tracked } from '@glimmer/tracking';
+import camelCase from 'lodash/camelCase.js';
+
+import isEmailValid from '../../../utils/email-validator.js';
+import isPasswordValid from '../../../utils/password-validator.js';
+
+const ERROR_INPUT_MESSAGE_MAP = {
+  firstName: 'components.authentication.signup-form.fields.firstname.error',
+  lastName: 'components.authentication.signup-form.fields.lastname.error',
+  email: 'components.authentication.signup-form.fields.email.error',
+  password: 'common.validation.password.error',
+  cgu: 'common.cgu.error',
+};
+
+class FirstName {
+  @tracked status = 'default';
+  @tracked errorMessage = null;
+
+  validate(value) {
+    return value ? true : false;
```

Comment:
```suggestion
    return Boolean(value);
```

Pareil pour le validate de `LastName` 😄